.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+|**Field**|**Value**|
+|:---|:---|
+|**_Submission Date_**| [DATE] |
+|**_Submitter_**| [USER] |
+|**_Originating Project/Creator_**| |
+|**_Base Branch_**| devel |
+|**_Short Description_**| |
+|**_Affected Component_**|  |
+|**_Affected Architectures(s)_**|  |
```

Comment:
Is this relevant for most changes? In the rare case where a particular architecture is affected I'd expect it to come up as a part of the change description.

Reply:
This is here because the team wanted a quick way of tracking changes in C++ code that affect certain architectures.

Reply:
Is a field in the template the best place to track this? Architecture specific changes are rare and because it's a field in the header it's not easy to search for. What about tagging to pull requests that affect specific architectures?

Reply:
This field header is really to make reviewing and testing quick.  The reason this field is there is so the following process can be done:

1. Look at architecture
2. Spin-up proper architecture
3. Follow testing recommendations

It isn't so much about searching, but specifically calling out if a change needs to be tested with specific hardware.